Sources/HTTPManager.swift

```diff
@@ -1620,6 +1620,10 @@ extension HTTPManager {
             switch uploadBody {
             case .data(let data)?:
                 networkTask = inner.session.uploadTask(with: urlRequest, from: data)
+            case .formUrlEncoded(let queryItems)?:
+                networkTask = inner.session.uploadTask(with: urlRequest, from: FormURLEncoded.data(for: queryItems))
+            case .json(let json)?:
+                networkTask = inner.session.uploadTask(with: urlRequest, from: JSON.encodeAsData(json))
```

Comment:
If you want to do this, you need to do it at the top of the method (where `.formUrlEncoded` is handled), otherwise you're doing a bunch of work inside of a critical section that doesn't need to be done there.